### ISSUE TITLE
RStudio crashes

### ISSUE THREAD

**Issue by pozsgaig** [rstudio/rstudio#1718]
I am experiencing some problems with RStudio Desktop now. I crashes whenever I try to open any interactive windows. It crashed immediately when I try to open Colour Picker, run this code
```
library('mindr')
example(markmap)
```
or this code
```
library(diffobj)
a <- b <- matrix(1:100, ncol=2)
a <- a[-20,]
b <- b[-45,]
b[c(18, 44)] <- 999
diffPrint(target=a, current=b)  
```
The crash seems to happen just after the call for a browser (listen to 127.0.01). I re-installed RStudio, R, updated all packages, tried to change default browser, disabled antivirus/firewall. No success so far.

Diagnostic report is not of a great help. Any ideas? Thnx

[1] "1.1.383"

$R
[1] "C:\\PROGRA~1\\R\\R-34~1.2PA\\bin\\x64\\R.exe"

$pdflatex
[1] "C:\\PROGRA~1\\MIKTEX~1.9\\miktex\\bin\\x64\\pdflatex.exe"

$bibtex
[1] "C:\\PROGRA~1\\MIKTEX~1.9\\miktex\\bin\\x64\\bibtex.exe"

$gcc
[1] ""

$git
[1] ""

$svn
[1] ""

R version 3.4.2 Patched (2017-11-02 r73659)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252    LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.2 tools_3.4.2    yaml_2.1.14   

SysInfo:
        sysname         release         version        nodename         machine           login            user  effective_user 
      "Windows"      ">= 8 x64"    "build 9200" "POZSIX_LAPTOP"        "x86-64"      "pozsgaig"      "pozsgaig"      "pozsgaig" 

R Version:
               _                                          
platform       x86_64-w64-mingw32                         
arch           x86_64                                     
os             mingw32                                    
system         x86_64, mingw32                            
status         Patched                                    
major          3                                          
minor          4.2                                        
year           2017                                       
month          11                                         
day            02                                         
svn rev        73659                                      
language       R                                          
version.string R version 3.4.2 Patched (2017-11-02 r73659)
nickname       Short Summer                               
$`__COMPAT_LAYER`
[1] "HighDpiAware DetectorsWin7"

$ALLUSERSPROFILE
[1] "C:\\ProgramData"

$AMDAPPSDKROOT
[1] "c:\\Program Files (x86)\\AMD APP\\"

$APPDATA
[1] "C:\\Users\\pozsgaig\\AppData\\Roaming"

$CLICOLOR_FORCE
[1] "1"

$CommonProgramFiles
[1] "C:\\Program Files\\Common Files"

$`CommonProgramFiles(x86)`
[1] "C:\\Program Files (x86)\\Common Files"

$CommonProgramW6432
[1] "C:\\Program Files\\Common Files"

$COMPUTERNAME
[1] "POZSIX_LAPTOP"

$ComSpec
[1] "C:\\windows\\system32\\cmd.exe"

$DISPLAY
[1] ":0"

$ESET_OPTIONS
[1] "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               "

$FP_NO_HOST_CHECK
[1] "NO"

$GFORTRAN_STDERR_UNIT
[1] "-1"

$GFORTRAN_STDOUT_UNIT
[1] "-1"

$HOME
[1] "C:/Users/pozsgaig/Documents"

$HOMEDRIVE
[1] "C:"

$HOMEPATH
[1] "\\Users\\pozsgaig"

$LOCALAPPDATA
[1] "C:\\Users\\pozsgaig\\AppData\\Local"

$LOGONSERVER
[1] "\\\\MicrosoftAccount"

$NUMBER_OF_PROCESSORS
[1] "4"

$OS
[1] "Windows_NT"

$PATH
[1] "C:\\Program Files\\R\\R-3.4.2patched\\bin\\x64;C:\\ProgramData\\Oracle\\Java\\javapath;C:\\Program Files (x86)\\PHP\\;c:\\Program Files (x86)\\AMD APP\\bin\\x86_64;c:\\Program Files (x86)\\AMD APP\\bin\\x86;c:\\Program Files (x86)\\Intel\\iCLS Client\\;c:\\Program Files\\Intel\\iCLS Client\\;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;c:\\Program Files (x86)\\ATI Technologies\\ATI.ACE\\Core-Static;C:\\Program Files (x86)\\Windows Live\\Shared;C:\\Program Files\\MiKTeX 2.9\\miktex\\bin\\x64\\;C:\\Program Files (x86)\\Bluefish;C:\\Program Files (x86)\\Java\\jre7\\bin\\;C:\\Program Files (x86)\\MySQL\\MySQL Server 5.5\\bin;C:\\windows\\system32;C:\\windows;C:\\windows\\System32\\Wbem;C:\\windows\\System32\\W;C:\\Program Files (x86)\\QuickTime\\QTSystem\\;C:\\Program Files\\Intel\\WiFi\\bin\\;C:\\Program Files\\Common Files\\Intel\\WirelessCommon\\;C:\\Program Files (x86)\\Skype\\Phone\\;C:\\Users\\pozsgaig\\AppData\\Local\\Programs\\Python\\Python35-32\\Scripts\\;C:\\Users\\pozsgaig\\AppData\\Local\\Programs\\Python\\Python35-32\\;C:\\ProgramData\\Oracle\\Java\\javapath;C:\\Program Files (x86)\\PHP\\;c:\\Program Files (x86)\\AMD APP\\bin\\x86_64;c:\\Program Files (x86)\\AMD APP\\bin\\x86;c:\\Program Files (x86)\\Intel\\iCLS Client\\;c:\\Program Files\\Intel\\iCLS Client\\;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;c:\\Program Files (x86)\\ATI Technologies\\ATI.ACE\\Core-Static;C:\\Program Files (x86)\\Windows Live\\Shared;C:\\Program Files\\MiKTeX 2.9\\miktex\\bin\\x64\\;C:\\Program Files (x86)\\Bluefish;C:\\Program Files (x86)\\Java\\jre7\\bin\\;C:\\Program Files (x86)\\MySQL\\MySQL Server 5.5\\bin;C:\\Program Files (x86)\\MySQL\\MySQL Utilities 1.3.4\\;C:\\windows\\system32;C:\\windows;C:\\windows\\System32\\Wbem;C:\\windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\QuickTime\\QTSystem\\;C:\\Program Files\\gs\\gs9.15\\bin\\;C:\\Program Files (x86)\\GmoteServer\\bin\\vlc;C:\\Program Files\\Intel\\WiFi\\bin\\;C:\\Program Files\\Common Files\\Intel\\WirelessCommon\\"

$PATHEXT
[1] ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC"

$PHPRC
[1] "C:\\Program Files (x86)\\PHP\\"

$PROCESSOR_ARCHITECTURE
[1] "AMD64"

$PROCESSOR_IDENTIFIER
[1] "Intel64 Family 6 Model 58 Stepping 9, GenuineIntel"

$PROCESSOR_LEVEL
[1] "6"

$PROCESSOR_REVISION
[1] "3a09"

$ProgramData
[1] "C:\\ProgramData"

$ProgramFiles
[1] "C:\\Program Files"

$`ProgramFiles(x86)`
[1] "C:\\Program Files (x86)"

$ProgramW6432
[1] "C:\\Program Files"

$PSModulePath
[1] "C:\\windows\\system32\\WindowsPowerShell\\v1.0\\Modules\\"

$PUBLIC
[1] "C:\\Users\\Public"

$R_ARCH
[1] "/x64"

$R_COMPILED_BY
[1] "gcc 4.9.3"

$R_DOC_DIR
[1] "C:/PROGRA~1/R/R-34~1.2PA/doc"

$R_HOME
[1] "C:/PROGRA~1/R/R-34~1.2PA"

$R_LIBS_USER
[1] "C:/Users/pozsgaig/Documents/R/win-library/3.4"

$R_USER
[1] "C:/Users/pozsgaig/Documents"

$RMARKDOWN_MATHJAX_PATH
[1] "C:/Program Files/RStudio/resources/mathjax-26"

$RS_LOCAL_PEER
[1] "\\\\.\\pipe\\23222-rsession"

$RS_RPOSTBACK_PATH
[1] "C:/Program Files/RStudio/bin/rpostback"

$RS_SHARED_SECRET
[1] "63341846741"

$RSTUDIO
[1] "1"

$RSTUDIO_CONSOLE_COLOR
[1] "256"

$RSTUDIO_CONSOLE_WIDTH
[1] "80"

$RSTUDIO_MSYS_SSH
[1] "C:/Program Files/RStudio/bin/msys-ssh-1000-18"

$RSTUDIO_PANDOC
[1] "C:/Program Files/RStudio/bin/pandoc"

$RSTUDIO_SESSION_PORT
[1] "23222"

$RSTUDIO_USER_IDENTITY
[1] "pozsgaig"

$RSTUDIO_WINUTILS
[1] "C:/Program Files/RStudio/bin/winutils"

$SESSIONNAME
[1] "Console"

$SystemDrive
[1] "C:"

$SystemRoot
[1] "C:\\windows"

$TEMP
[1] "C:\\Users\\pozsgaig\\AppData\\Local\\Temp"

$TERM
[1] "xterm-256color"

$TMP
[1] "C:\\Users\\pozsgaig\\AppData\\Local\\Temp"

$USERDOMAIN
[1] "POZSIX_LAPTOP"

$USERDOMAIN_ROAMINGPROFILE
[1] "POZSIX_LAPTOP"

$USERNAME
[1] "pozsgaig"

$USERPROFILE
[1] "C:\\Users\\pozsgaig"

$windir
[1] "C:\\windows"

 [1] ".GlobalEnv"        "tools:rstudio"     "package:stats"     "package:graphics"  "package:grDevices" "package:utils"     "package:datasets"  "package:methods"  
 [9] "Autoloads"         "package:base"

**Comment by kevinushey** [comment]
I wasn't able to reproduce this. I suspect this issue may be resolved either by updating your installed packages with e.g.

    update.packages(checkBuilt = TRUE)

or, perhaps, by cleaning up your `PATH` (it seems like you have a lot of entries there, and we have seen on very rare occasions that including directories containing libraries / binaries that conflict with RStudio can cause issues)

**Comment by pozsgaig** [comment]
As I said earlier, all the packages are updated. I checked the path but I do not really know what could I delete from there. I have the feeling shiny is causing the problems, shiny apps cause the same problem, colour picker brings up a shiny window, mindr is using shiny as well. Any more ideas? Thanks.

**Comment by jmcphers** [comment]
We haven't had any more reports or info on this in two years, and have never been able to repro it, so we're going to close it. If you're still seeing this with the latest version of RStudio (1.2.5019) and R packages, please feel free to reopen. Since this RStudio 1.1 issue was reported to be browser-related, there's a good chance it was fixed in 1.2, as it has a new browser engine.
